halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/AffinityConfig.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019 Snap, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.node;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class AffinityConfig {
+    public PodAffinity  podAntiAffinity;
```

Comment:
We generally just leave a single space between the type and variable name rather than align things like this.  (I realize style things like this aren't really well documented anywhere.)